### DISCUSSION TITLE
Failed Build - works locally, does not build on Vercel

### DISCUSSION THREAD

**Discussion by brokerxing** [vercel/community#4080] [Help]
### Summary

<html>
<body>

info  - Attempted to load @next/swc-linux-x64-gnu, but it was not installed
--
16:56:35.633 | info  - Attempted to load @next/swc-linux-x64-gnux32, but it was not installed
16:56:35.633 | info  - Attempted to load @next/swc-linux-x64-musl, but it was not installed
16:56:35.634 | error - Failed to load SWC binary for linux/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
16:56:35.657 | Error: Command "npm run build" exited with 1

</body>
</html>

Followed instructions in page re: SWC and still nothing is working.

### Steps to Reproduce

Not a lot to report here.  I can build the application on my local environment just fine.  Continues to fail when building on Vercel.

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share so we can help with debugging?
